std_msgs/Header header
string          final_result
string          partial_result
float32[]       spk_xvector
bool            is_speech_recognized    # unused